src/main/java/mekanism/client/gui/element/GuiScrollList.java

```diff
@@ -133,6 +130,8 @@ public int getScrollIndex() {
 
     @Override
     public void renderForeground(int xAxis, int yAxis) {
+        //TODO: For some reason the entire gui scroll list element gets its color shifted when GuiScreen#drawHoveringText is being drawn
+        // Resetting the color here does not seem to do anything to help fix this
```

Comment:
quite likely due to the hovering text method (forge's) ending with :
```
GlStateManager.enableLighting();
GlStateManager.enableDepth();
RenderHelper.enableStandardItemLighting();
GlStateManager.enableRescaleNormal();
```

`disableStandardItemLighting` might do the trick, possible `disableLighting` if it doesnt